OCS/Makefile

```diff
@@ -0,0 +1,5 @@
+.PHONY: all run
+all: run
```

Comment:
Is there a more descriptive target instead of "run"

Did you want to define a default?